README.md

```diff
@@ -36,3 +42,21 @@ cmake -G Ninja .. \
 ninja
 ninja check-mlir-clang
 ```
+
+#### Option 2: Using unified LLVM, MLIR, Clang, and Polygeist build
+
+Polygeist can also be built as an external LLVM project using [LLVM_EXTERNAL_PROJECTS](https://llvm.org/docs/CMake.html#llvm-related-variables).
+
+1. Build LLVM, MLIR, Clang, and Polygeist:
+```sh
+mkdir Polygeist/build
+cd Polygeist/build
+cmake -G Ninja ../llvm-project/llvm \
```

Comment:
Why do we build for all targets?

Reply:
I assume you mean the `ninja` command? I can remove it and just include the `ninja check-mlir-clang` command. That will built just the targets needed by `mlir-clang` and the tests.

Reply:
Sorry for the confusion, here I was referring to ```DLLVM_TARGETS_TO_BUILD```. To me, here it seems that we build all LLVM targets, while in "option 1" we build only for x86.

Reply:
Ah I see. I had copied the command to build polygeist based on an existing LLVM/MLIR, which didn't specify LLVM_TARGETS_TO_BUILD. I've added it to both examples, and updated it to "host" to avoid confusion if anyone tries to copy-paste the steps on a non-X86 machine.